styleguide-app/Examples/AssignmentIcon.elm

```diff
@@ -53,5 +53,20 @@ example =
                 [ ( "startPrimary", AssignmentIcon.startPrimary )
                 , ( "startSecondary", AssignmentIcon.startSecondary )
                 ]
+            , IconExamples.view "Activities"
+                [ ( "assessment", AssignmentIcon.assessment )
+                , ( "practice2", AssignmentIcon.practice2 )
```

Comment:
This name makes me wonder if there's a way to name these icons so that it's obvious from the name that they're categories of activity